pkg/madmin/info-commands.go

```diff
@@ -264,13 +264,14 @@ type ErasureBackend struct {
 
 // ServerProperties holds server information
 type ServerProperties struct {
-	State    string            `json:"state,omitempty"`
-	Endpoint string            `json:"endpoint,omitempty"`
-	Uptime   int64             `json:"uptime,omitempty"`
-	Version  string            `json:"version,omitempty"`
-	CommitID string            `json:"commitID,omitempty"`
-	Network  map[string]string `json:"network,omitempty"`
-	Disks    []Disk            `json:"drives,omitempty"`
+	State      string            `json:"state,omitempty"`
+	Endpoint   string            `json:"endpoint,omitempty"`
+	Uptime     int64             `json:"uptime,omitempty"`
+	Version    string            `json:"version,omitempty"`
+	CommitID   string            `json:"commitID,omitempty"`
+	Network    map[string]string `json:"network,omitempty"`
+	Disks      []Disk            `json:"drives,omitempty"`
+	PoolNumber string            `json:"PoolNumber,omitempty"`
```

Comment:
If it is a number why is it a string?

```suggestion
	PoolIndex int            `json:"pool_index"`
```

Reply:
PTAL @balamurugana ^^

Reply:
@klauspost I made the field as `int` and not changing the name. In other places, it is referred as pool number; not pool index.